rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RestClient.java

```diff
@@ -107,6 +107,17 @@ InputStream editNodeSource(@HeaderParam("sessionId") String sessionId,
             @FormParam("policyFileParameters") String[] policyFileParameters,
             @FormParam("nodesRecoverable") String nodesRecoverable);
 
+    @PUT
+    @Path("/rm/nodesource/parameter")
+    @Produces("application/json")
+    InputStream updateDynamicParameters(@HeaderParam("sessionId") String sessionId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)